git_machete/client.py

```diff
@@ -1667,12 +1666,13 @@ def handle_untracked_branch(
             new_remote: str,
             branch: LocalBranchShortName,
             is_called_from_traverse: bool,
+            is_called_from_create_pr: bool = False,
```

Comment:
I'd rather make this flag non-default (so that the caller always needs to specify the value explicitly)...